routes/web.php

```diff
@@ -19,14 +17,23 @@
 //Route::get('/', function () {
   //  return view('welcome');
 //});
-
+Route::redirect('/', '/homepage');
 Route::get('/homepage', function(){return view('homepage');})->name('homepage');
-Route::get('/booklist', function(){return view('booklist');})->name('booklist');
+Route::get("/books", [BookController::class,'show'])->name('books');
 
+Route::middleware(['auth'])->group(function(){
+  //CRUD
+  Route::get("/books/create",[BookController::class,'create'])->name('create');
+  Route::post("/books",[BookController::class,'store'])->name('book-create');
+  Route::put('books/{book}',[BookController::class,'update'])->name('book_update');
+  Route::get('/books/{book}/edit',[BookController::class,'edit'])->name('update');
+  Route::delete('/books/{book}',[BookController::class,'destroy'])->name('delete');
 
-Route::get("/adminpanel", [Admincontrol::class,'admin'])->name('adminpanel');
-Route::get("/logout",[Admincontrol::class,'logout'])->name('logout');
-Route::get("/AdminCrudInsert",[CrudOperator::class,'create'])->name('Insert');
-Route::post("/AdminCrudInsert",[CrudOperator::class,'bookcreate'])->name('BookCreate');
-Route::get("/AdminCrudUpdate",[CrudOperator::class,'update'])->name('Update');
-Route::get("/AdminCrudDelete",[CrudOperator::class,'delete'])->name('Delete');
+//Author
+  Route::get("/authors",[AuthorController::class,'show'])->name('authors');
```

Comment:
Semih'in attığı convention'a bakarak buradaki fonksiyon ismini "show" yerine "index" yapsak daha iyi olur.